docs/components/PrereleaseNotice.tsx

```diff
@@ -1,17 +1,29 @@
-import ReactMarkdown from 'react-markdown';
-
-import { mdComponents } from './plugins/api/APISectionUtils';
+import { PropsWithChildren } from 'react';
 
+import { Callout } from '~/ui/components/Callout';
 import { Collapsible } from '~/ui/components/Collapsible';
+import { A, CODE } from '~/ui/components/Text';
+
+type Props = PropsWithChildren<object>;
```

Comment:
```suggestion
type Props = PropsWithChildren;
```

Typing nit. We can also inline the type in that case, but was not able to suggest that.

Reply:
Ah okay, i missed that. Let me update this. Thanks!

Reply:
Done ✅